sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -22,6 +22,7 @@
 import argparse
 import json
 import logging
+import subprocess
```

Comment:
You may want to use https://github.com/apache/beam/blob/master/sdks/python/apache_beam/utils/processes.py instead.

Reply:
Done.